### PR TITLE
Fix admin workflows cypress test

### LINKED ISSUE #5370: 🐛 Bug - Fix admin workflows cypress tests on github
## 🐛 Bug

Fix breaking cypress tests on admin workflows. The tests are passing locally, but fail on GitHub actions.

## 🕵️ Details

I'm not sure why this is happening yet, but the problem seems to be cypress redirecting back to the users table (line 365 in image) with all the filters data from the `searchForUser()` command:
image

## ✅ Acceptance Criteria

- [ ] Tests pass locally and on github

### PR DESCRIPTION
**Pull request by yonikid15**
Resolves #5370

### CODE REVIEW

**Comment by yonikid15** [comment]
I'm not sure why this is happening yet, but the problem seems to be cypress redirecting back to the users table (line 365 in image) with all the filters data from the `searchForUser()` command:
image

**Comment by mnigh** [comment]
@yonikid15 can you create an issue and attach it to this PR so we can track what you are trying to fix?

File: apps/e2e/cypress/e2e/admin/admin-workflows.cy.js [hunk 1]
```diff
@@ -86,7 +82,7 @@ describe("Admin Workflow Tests", () => {
       .findByRole("row", { name: /applicant/i })
       .findByRole("link", { name: /Edit/i })
       .should("exist")
-      .and("be.visible")
+      .should('be.visible')
```
Comment (mnigh) [inline apps/e2e/cypress/e2e/admin/admin-workflows.cy.js:85]:
```suggestion
      .should("be.visible")
```

File: apps/e2e/cypress/e2e/admin/admin-workflows.cy.js [hunk 2]
```diff
@@ -126,8 +122,7 @@ describe("Admin Workflow Tests", () => {
     cy.findByRole("table")
       .findByRole("row", { name: /applicant/i })
       .findByRole("button", { name: /select/i })
-      .should("exist")
-      .and("be.visible")
+      .should('be.visible')
```
Comment (mnigh) [inline apps/e2e/cypress/e2e/admin/admin-workflows.cy.js:125]:
```suggestion
      .should("be.visible")
```

File: apps/e2e/cypress/e2e/admin/admin-workflows.cy.js [hunk 3]
```diff
@@ -137,4 +132,15 @@ describe("Admin Workflow Tests", () => {
 
     cy.verifyDownload('.csv', { contains: true });
   });
+
+  it("Tests fancy filter is present and triggers graphQL", () => {
```
Comment (mnigh) [inline apps/e2e/cypress/e2e/admin/admin-workflows.cy.js:136]:
```suggestion
  it("Opens filter dialog and triggers GraphQL query", () => {
```

File: apps/e2e/cypress/e2e/admin/admin-workflows.cy.js [hunk 4]
```diff
@@ -137,4 +132,15 @@ describe("Admin Workflow Tests", () => {
...
+    // find the applicant user to review
+    cy.findByRole("link", { name: /Manage users/i }).click();
+    cy.wait("@gqlAllUsersPaginatedQuery");
+
+    // tests fancy filter is present and triggers graphQL
```
Comment (mnigh) [inline apps/e2e/cypress/e2e/admin/admin-workflows.cy.js:141]:
```suggestion
```

**Review by mnigh — CHANGES_REQUESTED** [review]
Nice work @yonikid15! A few suggestions and one question but happy to see the e2e workflow is passing again.

File: apps/e2e/cypress/e2e/admin/admin-workflows.cy.js [hunk 5]
```diff
@@ -137,4 +132,15 @@ describe("Admin Workflow Tests", () => {
 
     cy.verifyDownload('.csv', { contains: true });
   });
+
+  it("Tests fancy filter is present and triggers graphQL", () => {
+    // find the applicant user to review
```
[the remark this answers is not in the document]
Comment (yonikid15) [reply apps/e2e/cypress/e2e/admin/admin-workflows.cy.js:137]: No this is needed, this takes the bot to the Users table before testing the dialog

**Review by mnigh — APPROVED** [review]
🚀

**Pull request merged**